config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -8,10 +8,10 @@ resources:
 
 images:
 - name: quay.io/confidential-containers/reqs-payload
-  newTag: latest
+  newTag: fad6ef4607e929c2d30d16dc8c62fbb29881a44c
```

Comment:
Don't we need https://github.com/confidential-containers/operator/pull/274 to be merged first before we take the reqs-payload release?

Reply:
hm good point. i guess the question is whether we can continue the release in parallel (i.e. merge this, tag the subprojects, and pick up the new reqs/operator changes when we do the next bundle update (step 13)). i am fine with waiting for #274, but that will serialize things a bit. wdyt @fidencio

Reply:
If the plan is to cycle back around and re-bump this once the operator PRs are finished then I'm okay with that, I just wanted to ensure that people were aware that we haven't finished yet, so some of this work will probably need to be redone.

Reply:
Yeah #280 also will change things. I think it is reasonable to pick up these changes later. It seems likely that a bundle that works with the current operator would also work with those PRs merged. Not ideal to have both things moving at once.

Reply:
I just merged this one, there are PRs coming on the Operator side; there may be PRs coming on the peer-pods side as well, but I'm fine on having kata-containers side of the things **not moing**.